src/ui/index.ts

```diff
@@ -1,19 +1,19 @@
-console.log('UI index.js loaded');
-
-import {Fin} from '../fin';
 import {ISenderInfo} from '../provider/Models/ISenderInfo';
 import {Notification} from '../Shared/Models/Notification';
-
-declare var fin: Fin;
+import {INotification} from './models/INotification';
+import {NotificationCenterAPI} from './NotificationCenterAPI';
+declare var window: Window&{notifications: NotificationCenterAPI};
 
 const IDENTITY = {
```

Comment:
Looks like we're using this in two places. Do we want a separate location for it so we don't have to worry about changing it in both spots?

Reply:
we may be able to reference a "shared" javascript file in both the UI and client, probably best for another smaller PR.